src/pages/SignUp/SignUp.js

```diff
@@ -1,47 +1,146 @@
+            alert('회원가입 성공');
+            history.push('/');
+          });
+      },
+      fail: function (err) {
+        alert(JSON.stringify(err));
+      },
+    });
+  };
+
+  // 이메일로 회원가입 및 로그인
+  const handleEmail = e => {
+    setEmail(e.target.value);
+  };
+
+  const handlePwd = e => {
+    setPwd(e.target.value);
+  };
+
+  const checkInfo = e => {
+    e.preventDefault();
+    if (checkEmail(email) && checkPwd(pwd)) {
+      fetch(SIGNUP_API, {
+        method: 'POST',
+        body: JSON.stringify({
+          email: email,
+          password: pwd,
+        }),
+      })
+        .then(res => res.json())
+        .then(res => {
+          if (res.accessToken) {
+            localStorage.setItem('accessToken', res.accessToken);
+            alert('회원가입 성공!');
+            history.push('/');
+          } else {
+            alert('회원가입에 실패하였습니다.');
+            setEmail('');
+            setPwd('');
+          }
+        });
+    } else {
+      alert('올바른 형식의 이메일과 비밀번호를 입력해주세요.');
+    }
+  };
+
   return (
     <Wrapper>
       <SignUpBox>
         <Title>회원가입</Title>
-        <div className="inputWrapper">
-          <Input placeholder="이메일" />
-          <Input placeholder="비밀번호" />
-        </div>
-        <div className="checkWrapper">
-          <Label>
-            <div className="border">
-              <input type="checkbox" name="agree" />
-              아래 내용에 모두 동의합니다.
-            </div>
-          </Label>
-          <Label>
-            <div>
-              <input type="checkbox" name="agree" />
-              [필수] 이용약관 동의
-            </div>
-            <div className="showMore">상세보기</div>
-          </Label>
-          <Label>
-            <div>
-              <input type="checkbox" name="agree" />
-              [필수] 개인정보 수집 이용 동의
-            </div>
-            <div className="showMore">상세보기</div>
-          </Label>
-        </div>
-        <ButtonWrapper>
-          <div>
-            <Button>이메일로 가입</Button>
+        <form onSubmit={checkInfo}>
+          <div className="inputWrapper">
+            <Input placeholder="이메일" type="text" onChange={handleEmail} />
+            <Input
+              placeholder="비밀번호"
+              type="password"
+              onChange={handlePwd}
+            />
           </div>
           <div>
-            <Button kakao>카카오로 로그인</Button>
+            <Label>
+              <div>
+                <input
+                  type="checkbox"
+                  name="checkAll"
+                  onChange={checkAll}
+                  checked={checkList.length === 2 ? true : false}
+                />
+                아래 내용에 모두 동의합니다.
+              </div>
+            </Label>
+            <Label>
+              <div>
+                <input
+                  type="checkbox"
+                  name="terms"
+                  onChange={handleCheck}
+                  checked={checkList.includes('terms') ? true : false}
```

Comment:
```suggestion
                  checked={checkList.includes('terms')}
```
includes method 자체가 boolean을 리턴해서 삼항 필요없을 것 같습니다!